src/module_TOUCHSCREEN/TouchscreenBackEnd.java

```diff
@@ -6,7 +6,7 @@
  * 
  */
 
-package module_TOUCHSCREEN;
```

Comment:
I kept it because the code in there was useful for helping me solve that problem, but it was only a resource. It's not necessary.